public/app/features/dashboard-scene/serialization/__snapshots__/transformSceneToSaveModelSchemaV2.test.ts.snap

```diff
@@ -2,7 +2,66 @@
 
 exports[`transformSceneToSaveModelSchemaV2 should transform scene to save model schema v2 1`] = `
 {
-  "annotations": [],
+  "annotations": [
+    {
+      "kind": "AnnotationQuery",
+      "spec": {
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --",
+        },
+        "enable": true,
+        "filter": {
+          "exclude": false,
+          "ids": [],
+        },
+        "hide": false,
+        "iconColor": "red",
+        "name": "query1",
+        "query": {
+          "kind": "grafana",
+          "spec": {
+            "datasource": {
+              "type": "grafana",
+              "uid": "-- Grafana --",
+            },
+            "enable": true,
+            "iconColor": "red",
+            "name": "query1",
+          },
+        },
+      },
+    },
+    {
+      "kind": "AnnotationQuery",
+      "spec": {
+        "datasource": {
+          "type": "prometheus",
+          "uid": "abcdef",
+        },
+        "enable": true,
+        "filter": {
+          "exclude": false,
+          "ids": [],
+        },
+        "hide": true,
+        "iconColor": "blue",
+        "name": "query2",
+        "query": {
+          "kind": "prometheus",
+          "spec": {
+            "datasource": {
+              "type": "prometheus",
+              "uid": "abcdef",
+            },
+            "enable": true,
+            "iconColor": "blue",
+            "name": "query2",
+          },
```

Comment:
This doesn't feel right. See the spec proposal in the design doc: https://docs.google.com/document/d/1H9cnQJqdcxYFi2juEt5CBkiSRnCr4paqznDBYiHCKqM/edit?tab=t.0#heading=h.1bs4nm6bogp2. `query: DataQuery` should not have information about the data source, that information is provided by higher level spec - in this case it's the `AnnotationQuery`